src/components/SaveExport/ExportDialog.jsx

```diff
@@ -164,6 +241,43 @@ export default () => {
                     <Col sm={8}>
                         <Card className="h-100">
                             <Card.Body>
+                                <h2>Area to export</h2>
+                                <ToggleButtonGroup
+                                    type="radio"
+                                    name="radio-export"
+                                    className="radio-export"
+                                    value={radioValue}
+                                >
+                                    {radios
+                                        .filter(
+                                            radio =>
+                                                radio.value !== 2 ||
+                                                cursorBegin != null
+                                        )
+                                        .map(radio => (
+                                            <ToggleButton
+                                                id={radio.id}
+                                                key={radio.id}
+                                                value={radio.value}
+                                                type="radio"
+                                                variant={
+                                                    radioValue === radio.value
+                                                        ? 'set'
+                                                        : 'unset'
+                                                }
+                                                checked={
+                                                    radioValue === radio.value
+                                                }
+                                                onChange={() =>
+                                                    updateRadioSelected(
+                                                        radio.value
+                                                    )
+                                                }
```

Comment:
Maybe change this to:

```suggestion
                                                onChange={() => {
                                                    setRadioValue(radio.value);
                                                    radio.onSelect();
                                                }}
```

Then you can also remove the `updateRadioSelected` function.

Reply:
I also use the function in line 181 and 183. It may be easy to understand what happens when you use `radio.onSelect();` here, but in the other place I think it's clearer with a function with an appropriate name.